templates/tutor.hbs

```diff
@@ -6,7 +6,7 @@
     <div class="tutor__content-inner">
 
       <div class="tutor__title">
-        <div class="tutor__title-inner" {{a11y_attrs_heading 'notify'}}>
+        <div class="tutor__title-inner" role="heading" aria-level="{{a11y_aria_level _attributes.data-adapt-id '@component+1' _ariaLevel }}">
```

Comment:
Can you add `_id: parentModel.get('_id'),` in here:
https://github.com/adaptlearning/adapt-contrib-tutor/blob/248bc0f948a027b0e398ad0ad3faf5998edbadc9/js/TutorModel.js#L19-L27

and change this to `_id` just to match the core pr?

```suggestion
        <div class="tutor__title-inner" role="heading" aria-level="{{a11y_aria_level _id '@component+1' _ariaLevel }}">
```